threadedcomments/templatetags/threadedcommentstags.py

```diff
@@ -365,7 +365,7 @@ def render(self, context):
             comments = FreeThreadedComment.objects.order_by('-date_submitted')[:self.num]
         else:
             comments = ThreadedComment.objects.order_by('-date_submitted')[:self.num]
-        context[self.context_name] = comments
+        context[self.context_name] = list(comments)
```

Comment:
Adding list was to fix some templatetag tests.

Reply:
interesting, why were the tests failing without `list()`?

Reply:
The template tag tests are just printing the response. Since `QuerySet` `__str__` method changed in django 1.11, we need to wrap this as a list.

Reply:
ah that `__str__` thing again. ok